packages/extension/src/components/pages/AddNewWallet/ConfirmPassword/ConfirmPassword.tsx

```diff
@@ -27,8 +28,8 @@ export const ConfirmPassword: FC<ConfirmPasswordProps> = ({ setPassword, onConfi
   }, []);
 
   const handleConfirmPassword = useCallback(() => {
-    const passwordValue = (document.getElementById('password') as HTMLInputElement).value;
     //TODO: See how to refactor this maybe just comparing with a password string from the context would be enough
+    let passwordValue = passwordRef.current?.value;
```

Comment:
I would replace this to a `const` as you never re-assign the variable later on in the code.